### PR TITLE
 Added support for IBM POWER8/9 architectures

### PR DESCRIPTION
**Pull request by pantaray**
Hello DLC team!

We're running IBM POWER E880 servers in our HPC cluster infrastructure and I tweaked DeepLabCut a bit to run on that architecture. I modified the package requirements (added platform specifiers) and created a new conda environment file for getting DLC on ppc machines. Maybe these changes are useful for others who want to use DeepLabCut on similar hardware.

All the best from Frankfurt,
Stefan

### CODE REVIEW

File: conda-environments/DEEPLABCUT_PPC.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: DEEPLABCUT_PPC
+channels:
+  - https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+  - powerai
+  - defaults
+  - conda-forge
+  - hcc
+dependencies:
```
Comment (MMathisLab) [inline conda-environments/DEEPLABCUT_PPC.yaml:7]: DLC is not a dependency?

Comment (pantaray) [reply conda-environments/DEEPLABCUT_PPC.yaml:7]: Hi! Sorry, my bad - removed it for testing. DLC is in again.

**Comment by MMathisLab** [comment]
Hey @pantaray after further thought, would you PR this into DLC Utils, whis are helper items for DLC; I think this would be best placed there, as we really try to limit to one (max 2 conda's), only due to major OS changes. There, you can also add a note about when and how to use it for the docs!

**Comment by pantaray** [comment]
Hi @MMathisLab Sure, no problem. Aside from the dev-ops modifications I had to introduce some platform-dependent changes to the default video codec in the corresponding helper routines (in `auxfun_videos.py` and `make_labeled_video.py`). Do you want me to create a patch for this and open a PR in DLCutils?

**Pull request closed without merging**